packages/oui-inline-adder/README.md

```diff
@@ -0,0 +1,153 @@
+# Inline adder
+
+<component-status cx-design="complete" ux="complete"></component-status>
+
+An inline Adder is used to present fields on a line. The special feature of the inline Adder is that you can add lines as needed. By default, only one line is displayed in the Inline Adder. When a new line is added, it becomes read-only and another new line is created below it.
+
+## Usage
+
+All form elements in an Inline Adder must be short. An Inline Adder can contain a maximum of 3 form elements.
+
+### Single fixed width field
+
+```html:preview
+<oui-inline-adder
+    on-add="$ctrl.addItem(item)"
+    on-remove="$ctrl.removeItem(item)">
+    <oui-inline-adder-item data-ng-repeat="item in $ctrl.singleFieldItems" item="item" is-new-item="item.isNew" aria-add-item="Add new item" aria-remove-item="Remove item {{$index+1}}">
+        <oui-inline-adder-row>
+            <div style="padding: 1rem; padding-bottom: 0;">
```

Comment:
No inline style. This should be in the ouiInlineAdderRow style

Reply:
This is done. It has been moved to ouiInlineAdderRow :)